### ISSUE TITLE
Log colors don’t display in Windows

### ISSUE THREAD

**Issue by bryanl** [ksonnet/ksonnet#570]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
>
> /kind feature

**What happened**:

Colors show control codes in Windows

**What you expected to happen**:

Colors should show or should not show control codes

**How to reproduce it (as minimally and precisely as possible)**:

Run `ks init app` and see the control codes

**Anything else we need to know?**:

**Environment**:
- ksonnet version (use `ks version`): 0.10.2

**Comment by redbaron** [comment]
AFAIK Windows10 cmd.exe got full color support in recent updates: https://blogs.msdn.microsoft.com/commandline/2016/09/22/24-bit-color-in-the-windows-console/

**Comment by bryanl** [comment]
Yes, this is true, but ksonnet still spits out color codes. I tried it with PowerShell and cmd.exe.

**Comment by redbaron** [comment]
hm, maybe color codes are enabled only when binaries run from WSL
